src/services/send.service.ts

```diff
@@ -133,23 +136,14 @@ export class SendService implements SendServiceAbstraction {
             if (sendData[0].type === SendType.Text) {
                 response = await this.apiService.postSend(request);
             } else {
-                const fd = new FormData();
                 try {
-                    const blob = new Blob([sendData[1]], { type: 'application/octet-stream' });
-                    fd.append('model', JSON.stringify(request));
-                    fd.append('data', blob, sendData[0].file.fileName.encryptedString);
+                    const uploadDataResponse = await this.apiService.postFileTypeSend(request);
+                    response = uploadDataResponse.sendResponse;
+
+                    this.fileUploadService.uploadSendFile(uploadDataResponse, sendData[0].file.fileName, sendData[1]);
                 } catch (e) {
-                    if (Utils.isNode && !Utils.isBrowser) {
-                        fd.append('model', JSON.stringify(request));
-                        fd.append('data', Buffer.from(sendData[1]) as any, {
-                            filepath: sendData[0].file.fileName.encryptedString,
-                            contentType: 'application/octet-stream',
-                        } as any);
-                    } else {
-                        throw e;
-                    }
+                    response = await this.legacyServerSendFileUpload(sendData, request);
```

Comment:
Do we want to try this based on the error response HTTP status code (i.e. "404") vs. just a blanket fallback for any error (what if it's a validation error, for example)?